### PR TITLE
Move derivation from web3State to userState

### PR DESCRIPTION
**Pull request by banciur**

### CODE REVIEW

File: app/config.ts [hunk 1]
```diff
@@ -65,22 +70,22 @@ function loadConfig(environment: object): IConfig {
 
       return {
         appState,
+        transactionSigning,
```
Comment (krzkaczor) [inline app/config.ts]: it's not needed here since we dont use web3 at all in this state

File: app/config.ts [hunk 2]
```diff
@@ -45,12 +46,16 @@ export function getRequiredValue(obj: any, key: string): string {
...
+    ...transactionSigningConfig,
+    defaultDerivationPath: getRequiredValue(environment, "DEFAULT_DERIVATION_PATH"),
+  };
 
   switch (appState) {
     case AppState.BEFORE_ANNOUNCEMENT:
       return {
         appState,
-        transactionSigning: transactionSigningConfig,
+        transactionSigning,
```
Comment (krzkaczor) [inline app/config.ts]: it's not needed here since we dont use web3 at all in this state

File: app/config.ts [hunk 3]
```diff
@@ -65,22 +70,22 @@ function loadConfig(environment: object): IConfig {
...
         announcedCfg: {
           startingDate,
         },
-        transactionSigning: transactionSigningConfig,
       };
     case AppState.CONTRACTS_DEPLOYED: {
       return {
         appState,
+        transactionSigning,
```
Comment (krzkaczor) [inline app/config.ts]: move it inside `contractsDeployed` key

File: app/reducers/userState.ts [hunk 4]
```diff
@@ -39,3 +50,7 @@ export function selectIsKnownUser(state: IUserState): boolean {
 export function selectLoading(state: IUserState): boolean {
   return state.loading;
 }
+
+export function selectDerivationPath(state: IUserState): string {
```
Comment (krzkaczor) [inline app/reducers/userState.ts]: this was used anywhere?

Comment (banciur) [reply app/reducers/userState.ts]: Not yet - it would be part of choosing account for user.

**Pull request merged**